data/weiphone

```diff
@@ -0,0 +1,3 @@
+feng.com
```

Comment:
multiple advertisement detected:
- https://yes1.feng.com/view.php?what=zone:xxx&n=ad414ae6xxx
- https://yes1.feng.com/frame.php?n=adxxxx&what=zone:xxxx
- ...

I suggest mark `yes1.feng.com` as advertisement.

Reply:
`feng-bbs-att-1255531212.image.myqcloud.com` seems to be the domain that weiphone is using for image storage. consider add it as well.

Reply:
tracker detected: `w.cnzz.com/c.php`.
consider creating another file to mark this.

Reply:
> `feng-bbs-att-1255531212.image.myqcloud.com` seems to be the domain that weiphone is using for image storage. consider add it as `full:` as well.

I'm always confused when encountering this kind of domains. The random number is likely to change.